client/lib/twilio.js

```diff
@@ -0,0 +1,32 @@
+module.exports = function sendInvite() {
+  const accountSid = 'AC16c88cb51171dd7a4b08dcd9c237ad3b';
+  const authToken = '8e916127cf5d4df8001d339cda26933b';
+  const client = require('twilio')(accountSid, authToken);
```

Comment:
Oh, you are calling `twilio` from the browser, not from the server. I would have done it from the server, but this is OK for demo purposes.

Reply:
this file should have been deleted, no longer needed!